.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": [
+        "airbnb-base",
```

Comment:
Prefer `airbnb` instead of `airbnb-base`.

Reply:
@loginov-rocks fixed